src/components/MessageListMessageCompact.vue

```diff
@@ -112,12 +112,12 @@ export default {
     position: absolute;
 }
 
-.kiwi-messagelist-message--compact.kiwi-messagelist-message-nick .kiwi-messagelist-time {
-    margin-right: 10px;
+.kiwi-messagelist-message--compact .kiwi-messagelist-nick:hover {
+    width: auto;
```

Comment:
Is this style not redundant as it's already set in MessageList.vue?

Reply:
Compact view overrides width default for the Nick - and sets a width on it, we then over ride that set width with auto.

ModernLayout uses the default width auto and doesn't over ride it however